synonymdev/bitkit-react-native
# fix(lightning): #987 lightning balances

### Description

- fix LN balances throughout the app.
- add claimable balance indicators to Assets & WalletDetail screen
- (quickfix) add unspendable punishment reserve balance indicator to Assets & WalletDetail screen
- add `useSwitchUnit` hook
- refactor `useBalance` & `getBalance`

### Linked Issues/Tasks

Closes https://github.com/synonymdev/bitkit/issues/987

### Type of change


### Tests


### Screenshot / Video

<img src="https://github.com/synonymdev/bitkit/assets/8538369/d9826abb-a324-4fe1-b44b-7f0f4b27a700" width="33%" />

### QA Notes

Make sure shown balances make sense throughout the app. Test all possible channel states: balanced, unbalanced, closing, opening.
